assembl/models/post.py

```diff
@@ -560,6 +560,10 @@ def restrict_to_owners(cls, query, user_id):
         "filter query according to object owners"
         return query.filter(cls.creator_id == user_id)
 
+    def is_bright_mirror_fiction(self):
+        parent_idea = self.indirect_idea_content_links_without_cache()
```

Comment:
You need to filter on `IdeaRelatedPostLink` only, so `parent_idea = [link.idea_id for link in content.indirect_idea_content_links_without_cache() if link.__class__.__name__ == 'IdeaRelatedPostLink']`